<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KEndingBalanceDlg</class>
 <widget class="QWizard" name="KEndingBalanceDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reconciliation Wizard</string>
  </property>
  <widget class="CheckingStartWizardPage" name="m_startPageCheckings">
  </widget>
  <widget class="PreviousPostponeWizardPage" name="m_pagePreviousPostpone">
  </widget>
  <widget class="CheckingStatementInfoWizardPage" name="m_statementInfoPageCheckings">
  </widget>
  <widget class="InterestChargeCheckingsWizardPage" name="m_interestChargeCheckings">
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
   <customwidget>
    <class>CheckingStartWizardPage</class>
    <extends>QWizardPage</extends>
    <header>checkingstartwizardpage.h</header>
   </customwidget>
   <customwidget>
    <class>CheckingStatementInfoWizardPage</class>
    <extends>QWizardPage</extends>
    <header>checkingstatementinfowizardpage.h</header>
   </customwidget>
   <customwidget>
    <class>InterestChargeCheckingsWizardPage</class>
    <extends>QWizardPage</extends>
    <header>interestchargecheckingswizardpage.h</header>
   </customwidget>
   <customwidget>
    <class>PreviousPostponeWizardPage</class>
    <extends>QWizardPage</extends>
    <header>previouspostponewizardpage.h</header>
   </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KMyMoneyDateInput</class>
   <extends>QFrame</extends>
   <header>kmymoneydateinput.h</header>
  </customwidget>
  <customwidget>
   <class>KMyMoneyCategory</class>
   <extends>KComboBox</extends>
   <header>kmymoneycategory.h</header>
  </customwidget>
  <customwidget>
   <class>AmountEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">amountedit.h</header>
  </customwidget>
  <customwidget>
   <class>KMyMoneyPayeeCombo</class>
   <extends>KComboBox</extends>
   <header>kmymoneypayeecombo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KMyMoneyAccountSelector</class>
   <extends>QWidget</extends>
   <header>kmymoneyaccountselector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
